src/dnlib.csproj

```diff
@@ -66,7 +67,7 @@ For better *Windows PDB* writer support, you should add a reference to `Microsof
     <DefineConstants>$(DefineConstants);TRACE</DefineConstants>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(OS)' != 'Windows_NT' ">
```

Comment:
This whole block can be removed now, we don't need to support building with mono.